packages/gitgraph-core/package.json

```diff
@@ -53,6 +53,9 @@
     "rollup": "^0.51.8",
     "uglify-es": "^3.3.9"
   },
+  "dependencies": {
+    "fastpriorityqueue": "^0.6.1"
```

Comment:
can't find a valid reason for importing this library

Reply:
@pvigier do you mind if we drop the `fastpriorityqueue` library? I have no preference, I was basically following your gitamine design.